frontend/src/Components/Map.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import { tileNames, tilesMap } from "./tilesMap.js";
+import "./Map.css";
+
+class Map extends React.Component {
+  shouldComponentUpdate() {
+    return false;
+  }
+
+  randomGrass() {
+    let randomNum = Math.floor(Math.random() * 40);
```

Comment:
Avoid hard-coded values like 40, 6, 23, 21: when I list them, you can't remember what it's about can you ?
Please use aptly-named variables instead ;-)

Reply:
We talked about it, at least use some continuous values!